test/MUXControlsTestApp/master/ListViewItem-7.xml

```diff
@@ -0,0 +1,56 @@
+[Windows.UI.Xaml.Controls.Grid]
+    Padding=0,0,0,0
+    CornerRadius=0,0,0,0
+    BorderThickness=0,0,0,0
+    BorderBrush=[NULL]
+    Background=[NULL]
+    Width=400
+    Margin=0,0,0,0
+    Height=400
+    FocusVisualSecondaryThickness=1,1,1,1
+    FocusVisualSecondaryBrush=#99FFFFFF
+    FocusVisualPrimaryThickness=2,2,2,2
+    FocusVisualPrimaryBrush=#FF000000
+    Visibility=Visible
+    RenderSize=400,400
+  [Windows.UI.Xaml.Controls.ListViewItem]
+      Padding=12,0,12,0
+      Foreground=#FF000000
+      BorderThickness=0,0,0,0
+      BorderBrush=[NULL]
+      Background=#00FFFFFF
+      CornerRadius=0,0,0,0
```

Comment:
Is there a difference between CornerRadius 0,0,0,0 and unset? Maybe we can have these "default" values not included to avoid differences between ListViewItem-5.xml and ListVIewItem-7.xml?

Reply:
CornerRadius doesn't exist before 7, but I guess there's no really difference between not exist and 0,0,0,0(default/unset). I'll remove the default value.

Reply:
Did this update happen yet?

Reply:
Would having values unset affect a dev who decides to set it later, or any potential future animated transitions between CornerRadius values on a control?

Reply:
@jevansaks  Yes, I already updated it but there have been a lot of visual changes recently it's hard to keep up with every change. You have to create 5 VMs and run tests on all of them to get the latest master files for each OS version. I wanted to try uploading the changed files to drop folder somewhere so we can just download them from pipeline test run.

@mdtauk When someone sets the value later it will show up in the file and they just need to diff the old and new versions to make sure that's the change they actually wanted, it won't prevent them from setting the values.